{
  assertThrows(log,new Callable<Object>(){
    @Override public Object call() throws Exception {
      return compute.execute(null,null);
    }
  }
,NullPointerException.class,"Ouch! Argument cannot be null: taskName");
}
